cnbinfo/readme.rst

```diff
@@ -0,0 +1,17 @@
+Description
+Is a python script that get all the usernames and ids for a given users list.
+First, we need to generate this user list file. You can generate it by running:
+  ./multirun.py --literal 'cat /etc/passwd' > users.txt
```

Comment:
Can we define these lines as code to improve readability?

Reply:
Can you please specify where to find multirun script?

Reply:
Can you show an example of how the users.txt file should look like?

Reply:
Done